### DISCUSSION TITLE
Open task list with direct float preview

### DISCUSSION THREAD

**Discussion by yngwi** [stevearc/overseer.nvim#258] [Q&A] [answered]
Hi, I wonder if it is possible to create a mapping that opens either the task list and the preview of the most recent task at the same time (or just the preview without having to go to the task list to have a quick peek to the full output?

**Comment by stevearc** [comment, marked as answer]
You could do `:OverseerQuickAction open float` to open the most recent task output in a float, but what I often do is use `:OverseerToggle bottom` to toggle a view at the bottom of the task list and task output.

**Reply by yngwi** [reply]
thank you
